README.md

```diff
@@ -1,31 +1,19 @@
 # Clustering Systems
 
 ## Running the clustering on OpenShift
```

Comment:
again, keep this section as is and create a new section for the experiment.

Reply:
Done :)